AirCasting/URLProviders/UserDefaultsBaseURLProvider.swift

```diff
@@ -12,6 +12,25 @@ class UserDefaultsBaseURLProvider: BaseURLProvider {
             userDefaults.url(forKey: "baseURL") ?? URL(string: "http://aircasting.org/api")!
         }
     }
+    
+    var baseAppURLwithoutPort: URL {
```

Comment:
like previously - I'm not sure this is needed. Why not single URL with port?

Reply:
Now it's obvious, when making I haven't thought about this